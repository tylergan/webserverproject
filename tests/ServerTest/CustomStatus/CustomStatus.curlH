curl -I localhost:8070/cgibin/custom_status.py