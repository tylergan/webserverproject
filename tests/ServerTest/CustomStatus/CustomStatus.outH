HTTP/1.1 222 A Custom Status
Content-Type: text/html

