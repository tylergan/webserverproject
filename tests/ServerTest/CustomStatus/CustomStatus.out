<h1>Other things which should only be included in the data body</h1>
